packages/plugin-ext/src/plugin/languages/rename.ts

```diff
@@ -57,7 +57,7 @@ export class RenameAdapter {
             if (rejectReason) {
                 return <WorkspaceEditDto>{
                     rejectReason,
-                    edits: undefined!
+                    edits: []
```

Comment:
this shouldn't break with the expected handling of `rejectReason`, which BTW is commented with a todo to be a proper rejection in the origin of this.